packages/react-internal/src/components/Autofill/Autofill.tsx

```diff
@@ -356,5 +356,18 @@ function _doesTextStartWith(text: string, startWith: string): boolean {
   if (!text || !startWith) {
     return false;
   }
+
+  if (process.env.NODE_ENV !== 'production') {
+    for (const val of [text, startWith]) {
+      if (typeof val !== 'string') {
+        throw new Error(
+          `${
+            Autofill.name
+          } received non-string value(s) of type ${typeof val} from either input's value or suggestedDisplayValue`,
```

Comment:
I think it would be better if the error message included the actual value received in addition to the type, and which specific prop had the invalid value.

Also sorry if I missed this in the issue but do we know that both of these are always expected to be strings?

Reply:
> I think it would be better if the error message included the actual value received in addition to the type, and which specific prop had the invalid value.

Thanks, I will look into this

>Also sorry if I missed this in the issue but do we know that both of these are always expected to be strings?

I think these may help following the issue: 
- https://github.com/microsoft/fluentui/issues/15178#issuecomment-765122975
- https://github.com/microsoft/fluentui/issues/15178#issuecomment-765686718 

We know that these are expected to be strings. This code block was added to handle edge cases in the client application that somehow manages to pass non-string values. This throws development error to warn such cases.